lang/string_extractor/parsers/examine_action.py

```diff
@@ -0,0 +1,14 @@
+from ..write_text import write_text
+from .effect_on_condition import parse_effect_on_condition
+
+
+def parse_examine_action(json, origin):
+    if "query_msg" in json:
+        write_text(json["query_msg"], origin, comment="Query message")
```

Comment:
It would be more helpful to translators if the terrain or furniture name is available in the translation comment.

Reply:
Added.